### PR TITLE
Added plan of action for offending parties, as well as some things from our old CoC

### PR DESCRIPTION
**Pull request by cocoa1231**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ Be respectful
 
 Respect yourself, and respect others. Be courteous to those around you. If someone indicates they don’t wish to be photographed, respect that wish. If someone indicates they would like to be left alone, let them be. Our event venues and online spaces may be shared with members of the public; please be considerate to all patrons of these locations.
 
+Before Direct Messaging someone, ask for their permission on the group. If they allow you to, only then proceed to do so. And remember that LinuxChix India is not a dating service. The people in here are here to learn, explore and socialze in a professional environment.
```
Comment (CuriousLearner) [inline README.md]: I will be able to add this if you remove this sentence `And remember that LinuxChix India is not a dating service.`

File: README.md [hunk 2]
```diff
@@ -58,6 +60,11 @@ Here are some examples of behaviours that have occurred at past events which are
     - sexist, racist, or other exclusionary jokes;
     - unwarranted exclusion from conference or related events based on age, gender, sexual orientation, disability, physical appearance, body size, race, religion;
 
+Any violation of the Code of Conduct will be dealth with a three step protocol.
```
Comment (CuriousLearner) [inline README.md]: I wouldn't suggest adding these limitations. Code of conduct violations should be treated at the discretion of admins. It is not always necessary to give a warning first, that, of course, depends on the sensitivity of the report.

What do you think?

**Review by CuriousLearner — COMMENTED** [review]
Hello Jatin,

Thanks for your PR. Since this is a repo to be used by local communities, I will like to avoid any information that isn't helpful for everyone and specifically the name of the community.

I encourage you to fork this in the organization and modify as you see appropriate depending on the LICENSE.

I will be able to accept some part of the changes, as mentioned inline. Let me know how it sounds.

File: README.md (hunk 2 above)
Comment (cocoa1231) [reply README.md]: Yes, seeing as this is a general CoC, I agree that it should be up to the mods to figure out how to deal with violations. I'll remove this

File: README.md (hunk 1 above)
Comment (cocoa1231) [reply README.md]: Yep, makes sense. Did it

**Comment by cocoa1231** [comment]
Oh okay, I'll fork this to LinuxChixIn then. And I've made the changes you requested Thanks!

**Review by CuriousLearner — APPROVED** [review]
LGTM 🌮

**Pull request merged**